src/aave-v3/SupplyVault.sol

```diff
@@ -189,6 +189,16 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
         super._withdraw(_caller, _receiver, _owner, _assets, _shares);
     }
 
+    function _beforeTokenTransfer(
+        address _from,
+        address _to,
+        uint256 _amount
+    ) internal virtual override {
+        _accrueUnclaimedRewards(_from);
+        _accrueUnclaimedRewards(_to);
+        super._beforeTokenTransfer(_from, _to, _amount);
```

Comment:
only the supplyvault has been updated with that no?

Reply:
yes, but I think that we should remove the supply harvest vaults anyways

Reply:
Shit I wanted to say Aave V3, why not compound?

Reply:
It has already been fixed on Compound (hot fix in prod)

Reply:
not that super._beforeTokenTransfer(_from, _to, _amount);

Reply:
btw harvest vaults are already removed 🙃

Reply:
I think we can remove this line since it's just a virtual function without logic:
<img width="334" alt="Screenshot 2022-12-02 at 11 57 52" src="https://user-images.githubusercontent.com/44097430/205277646-f0fcb3d6-b652-4a43-8631-b25b561a4d64.png">

Reply:
>btw harvest vaults are already removed 🙃 

not on main at least ..

Reply:
Everything is in upgrade-0 yes